components/ScrimDelete.tsx

```diff
@@ -24,9 +32,12 @@ export const ScrimDelete = ({
   header,
   warningMessage,
   confirmMessage,
+  checkboxMessage,
   error,
   isLoading,
 }: ScrimDelete) => {
+  const [checked, setChecked] = useState(false);
```

Comment:
I'd go with `isChecked` and `setIsChecked` as this is a common naming convention for boolean variables.

Reply:
EDS uses "checked" and it's a standard property for HTML elements, so in this project I think it works better for consistency